.env.example

```
@@ -1,3 +1,5 @@
 POSTGRES_DB=
 POSTGRES_USER=
-POSTGRES_PASSWORD=
\ No newline at end of file
+POSTGRES_PASSWORD=
+RABBITMQ_USERNAME=
+RABBITMQ_PASSWORD=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)